feat: add node selectors to connection settings

samsiegart opened the pull request:
fixes https://github.com/Agoric/wallet-app/issues/130

Adds auto-suggest inputs for rpc and api nodes. The suggestions are fetched from the currently selected network config. When the inputs are empty, it just defaults to a random node, so the inputs are not required.

## Screenshot

![image](https://github.com/Agoric/wallet-app/assets/8848650/6f25c996-c1ef-4913-b971-03ec0a3c9287)

0xpatrickdev reviewed (changes requested):
- Changing Localhost API URL seems to throw 
- “Testnet” is not mapping to anything - should this be Emerynet? 
- Localhost uses `origin` instead of `localhost` - is this intentional? 

https://github.com/Agoric/wallet-app/assets/11021913/32d10838-12c2-4014-9cf7-836f22c0b18f

samsiegart commented:
> Changing Localhost API URL seems to throw
Fixed, it was throwing if apiAddrs (or rpcAddrs) was missing from the network config.

> “Testnet” is not mapping to anything - should this be Emerynet?
Yea I think we used to have a "testnet" but not anymore, seems sensible to change to emerynet, just did.

> Localhost uses origin instead of localhost - is this intentional?
Yea, the wallet app itself has a network config so you can edit it if you're running the UI locally, I just changed it to "Local" instead of "Localhost" though so it's not as potentially misleading. The "Localhost" I think implied that your chain was running on localhost, not the network config.
